draft-ietf-tls-esni.md

```diff
@@ -51,6 +51,11 @@ encrypting a ClientHello message under a server public key.
 
 # Introduction
 
+[[TODO: Add a high-level description to this section of the security goal of
```

Comment:
Agreed. Let's make an issue for this (and remove the TODO)!

Reply:
Done. https://github.com/tlswg/draft-ietf-tls-esni/issues/260